src/BaseTemplates/WebAPI/project.json

```diff
@@ -36,7 +36,10 @@ $if$ ($context$ == WebCore) $else$
   },
 $endif$
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
```

Comment:
I'll update the import to the same as the razor one then for consistency.

- Phil


On Apr 18, 2016, at 4:34 PM, Pawel Kadluczka <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/BaseTemplates/WebAPI/project.json<https://github.com/aspnet/Templates/pull/512#discussion_r60152671>:

> @@ -36,7 +36,10 @@ $if$ ($context$ == WebCore) $else$
>    },
>  $endif$
>    "tools": {
> -    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
> +    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
> +      "version": "__AspNetCorePlatformVersion__",
> +      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"


@phenning<https://github.com/phenning> - yes. These are for Json.NET<http://json.net>

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Templates/pull/512/files/46c506bff95b7d383c07df3155e22abbe927eaa0#r60152671>